### PR TITLE
Use constant for configuring limit

### PR DESCRIPTION
**Pull request by tflhyl**
Avoid confusion e.g. #14.

### CODE REVIEW

File: Example/Pickle/CarousellImagePickerController.swift [hunk 1]
```diff
@@ -47,7 +49,7 @@ internal class CarousellImagePickerController: ImagePickerController {
             )
 
             let subtitle = NSAttributedString(
-                string: "You can choose up to 4 photos for your listing.\n",
+                string: "You can choose up to \(selectionsLimit) photos for your listing.\n",
```
Comment (bcylin) [inline Example/Pickle/CarousellImagePickerController.swift]: Looks like a good idea! Does it need a `.stringsdict` in the example for both singular and plural photos?

Comment (ErAshu) [reply Example/Pickle/CarousellImagePickerController.swift]: How to pass `selectionsLimit` and hint text from viewController?

**Comment by ErAshu** [comment]
How to pass selectionsLimit and hint text from viewController?

**Comment by bcylin** [comment]
`ImagePickerSelection.limit(to: selectionsLimit)` is part of [`ImagePickerConfigurable`](https://carousell.github.io/pickle/Protocols/ImagePickerConfigurable.html#/s:6Pickle23ImagePickerConfigurableP17allowedSelectionsAA0bC9SelectionOSgv). It's passed to the image picker controller via [`init(selectedAssets:configuration:)`](https://carousell.github.io/pickle/Classes/ImagePickerController.html#/s:6Pickle21ImagePickerControllerCACSaySo7PHAssetCG14selectedAssets_AA0bC12Configurable_pSg13configurationtcfc).

Hint text is set via the [`hint`](https://carousell.github.io/pickle/Classes/ImagePickerController.html#/c:@M@Pickle@objc(cs)ImagePickerController(py)hint) attributed string.

**Pull request merged**